dpdk_spdk.md

```diff
@@ -35,7 +35,5 @@ Some application, written with the x86 architecture in mind, set the active dpdk
 
 ## Known issues
 
-* **rte_eth_stats_get:** The function rte_eth_stats_get does not work properly on C6gn instances. Applications, such as testpmd, that use this function to report statistics will report 0 instead of actual packet counts. This is a known issue and will be addressed soon.
```

Comment:
with any version of dpdk or is there a certain dpdk min version?

Reply:
I tested with DPDK 21.08.